Disable hanging test
Bug: https://github.com/zeromq/pyzmq/issues/1202

Patch by Marius Brehler <marbre@linux.sungazer.de>
--- pyzmq-17.1.0/zmq/tests/test_message.py
+++ pyzmq-17.1.0/zmq/tests/test_message.py
@@ -14,6 +14,7 @@ import time
 from pprint import pprint
 from unittest import TestCase
 
+import pytest
 import zmq
 from zmq.tests import BaseZMQTestCase, SkipTest, skip_pypy, PYPY
 from zmq.utils.strtypes import unicode, bytes, b, u
@@ -262,6 +263,7 @@ class TestFrame(BaseZMQTestCase):
             self.assertEqual(s,r)
         self.assertEqual(s, m.bytes)
     
+    @pytest.mark.skip("test sometimes hangs")
     def test_buffer_numpy(self):
         """test non-copying numpy array messages"""
         try:
